icpc-wf/ansible/hosts

```diff
@@ -4,8 +4,9 @@ ansible_python_interpreter=/usr/bin/python3
 EFI_ORDER='2,1'
 
 [domserver]
-domjudge-primary     ansible_host=10.3.3.216 KEEPALIVED_PRIORITY=100 EFI_ORDER='0\,1\,3\,4'
-domjudge-backup      ansible_host=10.3.3.217 KEEPALIVED_PRIORITY=99 EFI_ORDER='0\,1\,3\,4'
+domjudge-primary ansible_host=domjudge.bapc.eu KEEPALIVED_PRIORITY=100
```

Comment:
tab is missing.

Reply:
And maybe that added line is not needed because its a leftover of BAPC?

Reply:
This file shouldn’t be committed at all